bson/src/main/org/bson/types/ObjectId.java

```diff
@@ -197,7 +147,9 @@ public ObjectId(final Date date, final int counter) {
      * @param processIdentifier the process identifier
      * @param counter           the counter
      * @throws IllegalArgumentException if the high order byte of machineIdentifier or counter is not zero
+     * @deprecated Use {@link #ObjectId(Date, int)} instead
      */
+    @Deprecated
```

Comment:
I have deprecated any constructors that have `machineIdentifier` or `processIdentifer` as parameters.